server/config/config.go

```diff
@@ -1042,6 +1042,7 @@ var DefaultSchedulers = SchedulerConfigs{
 	{Type: "balance-leader"},
 	{Type: "hot-region"},
 	{Type: "split-bucket"},
+	{Type: "transfer-leader"},
```

Comment:
How about promote-witness or something else, the current name sounds like the balance leader.

Reply:
Sure, thanks!